### DISCUSSION TITLE
bug: recent directory and file in project is not found in "go to files"

### DISCUSSION THREAD

**Discussion by wesinator** [community/community#112183] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

At the time that I write this, the committed file https://github.com/microsoft/winfile/blob/master/build/Winfile.props and anything else in the `build` directory is not available in "Go to file" on https://github.com/microsoft/winfile , and when I try to edit this file from it's page, it leads to a 404. 
![image](https://github.com/community/community/assets/5124946/982db8de-2358-4f9c-9ea2-8c68d05c5d41)

![image](https://github.com/community/community/assets/5124946/70ef33b8-3100-4bc7-bf7d-6c36f51ad8be)

I believe this is a significant bug/regression. I'm wondering if it has to do with `build` in the path becoming a restricted word somewhere on the backend. The `build` directory and file in question was newly added a few weeks ago in a co-authored commit, and updated twice.

also fwiw, in general I've noticed a lot of workflow issues in the Github UI with editing, forks, gists and PRs that weren't previously issues. there seems to be a lot of regressions in recent months.

**EDIT:** the PR issue occurred because I had a secondary fork (fork of a fork) that was archived. I expected it to create a new fork of the target one, but it's an edge case.

The issue with recent files not showing up in the repo "go to files" seems to be a problem possibly related to the recent incidents.

[NO_TRAIN]::

**Comment by look** [comment, marked as answer]
@wesinator there is a list of directories that are excluded from the fuzzy file matching and, unfortunately, `build` is one of them. This is because it usually contains build artifacts.
